src/codewars/m_sulkouski/github/com/MorseCodeDecoder.java

```diff
@@ -9,13 +9,16 @@
 //        Your task is to implement a function that would take the morse code as input and return a decoded human-readable string.
 
 
+import morse_code_decoderii.codewars.m_sulkouski.github.com.MorseCodeDecoder_II;
+
 public class MorseCodeDecoder {
     public static void main(String[] args) {
-        String decodedMessage = decode("   .   .");
+        String decodedMessage = decode("···· · −·−− ·−−− ··− −·· ·");
         System.out.println(decodedMessage);
     }
 
     public static String decode(String codedText) {
+        codedText = (codedText);
         String[] codedPhrase = codedText.trim().split("   ");
```

Comment:
to improve readability and constants reusability "   " could be a constant:
`final String SPACE = "   "`